submissions/cumiOyemike/php/weather.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+$temperature;
```

Comment:
No need for the code in line 3

Reply:
Hi, Ada. I just made the change. Kindly review again. Thank you.